### PR TITLE
Docker improvements

### PR DESCRIPTION
**Pull request by Artanicus**
Use supervisord to run both plotting & archiving by default. This can easily be overridden by volume mounting a replacement `/srv/supervisord.conf`

- Suggest default tagging the image as `latest` since it's the Docker default and often has a special meaning in orchestration.
- Update Chia reference to latest stable
- Avoid clobbering $UID which can have unintended consequences

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN /bin/bash ./install.sh
 
 COPY . /plotman
 
-RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman && deactivate"] 
+RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman && pip install supervisor && deactivate"]
```
Comment (altendky) [inline docker/Dockerfile:20]: We are already installing into an existing env which pip isn't the best at.  But we can still let pip know all the additional things it needs to install at once.

```suggestion
RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman supervisor && deactivate"]
```

File: build-docker-plotman.sh [hunk 2]
```diff
@@ -2,22 +2,20 @@
...
 BASE_CONTAINER="ubuntu:20.04"
-CHIA_GIT_REFERENCE="1.1.7"
+CHIA_GIT_REFERENCE="1.2.3"
 
 # The UID/GID should match the 'chia' owner of the directories on the host system
-UID=10001
-GID=10001
-
-docker rmi ${LOCAL_REGISTRY}/${PROJECT}:${TAG}
+bUID=10001
```
Comment (altendky) [inline build-docker-plotman.sh:10]: Not clobbering `UID` sounds good.  Does the `b` refer to anything in particular?

**Review by altendky — COMMENTED** [review]
Just a couple details here from me, for now.  @graemes, want to chime in here?

File: build-docker-plotman.sh (hunk 2 above)
Comment (Artanicus) [reply build-docker-plotman.sh:10]: Just build_UID essentially. I'll make it more descriptive :-)

**Comment by graemes** [comment]
Looks good to me.

**Comment by altendky** [comment]
Thanks for this and my apologies for the extended delay in handling it.

**Pull request merged**